main.js

```diff
@@ -4,19 +4,194 @@ const navClose = document.getElementById('close-menu');
+    feature: 'A page that uses all the inpt types, and filters for them ina good shape and send results.',
+    image: './images/project/images/survey.jpg',
+    technologies: ['HTML', 'inline CSS'],
+    liveVersionLink: 'https://andrelau11.github.io/Survey/',
+    sourceLink: 'https://github.com/AndreLau11/Survey',
+  },
+  {
+    id: 4,
+    name: 'Personal Portfolio',
+    description:
+      'A general skill termination that includes all I have been capable of doing.',
+    feature: 'A basic website',
+    image: './images/project/images/personalPortfolio.jpg',
+    technologies: ['HTML', 'inline CSS'],
+    liveVersionLink: 'https://andrelau11.github.io/Personal_Portfolio/',
+    sourceLink: 'https://github.com/AndreLau11/Personal_Portfolio',
+  },
+  {
+    id: 5,
+    name: 'My Product',
+    description:
+      'A website that gives out and promotes the product to get it sell on the market.',
+    feature: 'A publicitary page that shows the best sides of the product',
+    image: './images/project/images/myProduct.jpg',
+    technologies: ['HTML', 'inline CSS'],
+    liveVersionLink: 'https://andrelau11.github.io/My_Product/',
+    sourceLink: 'https://github.com/AndreLau11/My_Product',
+  },
+];
+
+function LoadProjects() {
+  let counter;
+  for (counter = 0; counter < projectInformation.length; counter += 1) {
+    // if (counter == 0) {
+    //   projectInformation[counter].id = 1;
+    // }
+    // let k;
+    // let technologies = '';
+    // for (k = 0; k < projectInformation[counter].technologies.length; k += 1) {
+    //   technologies += `<li>
+    //     <a href='#'>${projectInformation[counter].technologies[k]}</a>
+    //   </li>`;
+    // }
+
+    cardContainer.innerHTML += `<div class="projectsection${[counter]}">
+        <h2 class="projecttitle">${projectInformation[counter].name}</h2>
+        <p class="projectparagraph">${projectInformation[counter].description}</p>
+        <ul class="skill">
+            <li><a href=""><img class="skill-image-tag" src="./images/project/Tag0.png"
+                        alt="Rubi legend, as reference of the back end language used on the project"></a>
+            </li>
+            <li><a href=""><img class="skill-image-tag" src="./images/project/Tag1.png"
+                        alt="Bootstrap legend, as reference of the tool used for styling the page"></a></li>
+            <li><a href=""><img class="skill-image-tag" src="./images/project/Tag2.png"
+                        alt="HTML legend, as reference of the coding used on the front end"></a></li>
+        </ul>
+        <button data-target="${projectInformation[counter].id}" type="submit" class="projectbutton">See Project</button>
+    </div>`;
+  }
+
+  function hideModal() {
+    popUp.innerHTML = '';
+    popUp.classList.add('hidden');
+  }
+
+  function showModal(button) {
```

Comment:
[OPTIONAL] It would better to stop the page from scrolling after opening the pop-up for a better user experience.

Reply:
@medaminedev66 would you be  so kink of letting me know you meant to stop the pop up scroll or the main page scroll, thanks.

Reply:
@medaminedev66 would you be so kind of provide me a guideline I've been trying to find it and nothing, I've tried multiple ways and is just now working ,thank you